package.json

```diff
@@ -38,9 +38,10 @@
     "test": "react-scripts test",
     "eject": "react-scripts eject",
     "eslint": "eslint .",
-    "eslint:fix": "eslint --fix .",
-    "format": "prettier --write \"**/*.+(js|jsx|json|yml|yaml|css|md|vue)\"",
-    "scsslint": "sass-lint-auto-fix"
+    "eslint:fix": "eslint --fix --max-warnings=0",
+    "format": "prettier --write \"**/*.+(js|jsx|json|yml|yaml|md)\"",
+    "scss-lint": "sass-lint 'src/**/*.scss' -v -f table --max-warnings 328",
+    "scss-lint:fix": "sass-lint-auto-fix 'src/**/*.scss' -d -h"
```

Comment:
@KrishnaPravin from the docs of `sass-lint-auto-fix`

> By default, sass-lint will look for a .sass-lint.yml or .sasslintrc, or an entry in your package.json. When applying the resolver to a given rule, configuration is parsed from one of the above files..
> https://www.npmjs.com/package/sass-lint-auto-fix#configuring-which-rules-to-run-against

There is no mention of a file named `.sass-lint-auto-fix.yml` to be considered automatically as config. And I did not the see that file provided with `-c` option to the `sass-lint-auto-fix` command.

Reply:
We can enable/disable the resolvers using `sass-lint-auto-fix`. For eg: `no-css-comments` this resolver is disabled using `sass-lint-auto-fix.yml` but a warning is enabled in `sass-lint`.

Reply:
@KrishnaPravin I got it. But the file `sass-lint-auto-fix.yml` is not given to the command. Should not we have to do something like below?
```
"scss-lint:fix": "sass-lint-auto-fix 'src/**/*.scss' -d -h -c .sass-lint-auto-fix.yml"
```

Reply:
I will add `-c ` option

Reply:
I've added it